src/Api/SitesApi.php

```diff
@@ -69,4 +69,21 @@ public function removeCustomCertificate($siteId)
             'site_id' => $siteId,
         ]);
     }
+    /**
```

Comment:
could you put newlines around each function/docblock block? the formatter fixer doesn't seem to pick up on this unfortunately